lib/xcp_coverage.mli

```diff
@@ -0,0 +1,13 @@
+(* the build system chooses either 'coverage/enabled.ml' or
+ * 'coverage/disabled.ml' as an implementation *)
+
```

Comment:
If I understand this aright, any executable that uses this needs to make exactly one call to exactly one of `init` and `dispatcher_init`.
If so, let's write that down.